File: leetcode_771/solution.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        return sum(s in J for s in S)


if __name__ == '__main__':
    J = 'aA'
    S = 'aAAbbbb'
    ret = Solution().numJewelsInStones(J, S)
    print(ret)

File: leetcode_771/solution.go
package main

import (
	"fmt"
	"strings"
)

func numJewelsInStones(J string, S string) int {
	sum := 0
	for _, j := range J {
		sum += strings.Count(S, string(j))
	}
	return sum;
}

func main()  {
	J := "aA"
	S := "aAAbbbb"
	ret := numJewelsInStones(J, S)
	fmt.Println(ret)
}